sql_schema/Ecommerce.sql

```diff
@@ -0,0 +1,68 @@
+CREATE DATABASE Ecommerce;
+
+USE Ecommerce;
+
+CREATE TABLE Customer(
+CustomerID INT PRIMARY KEY,
+CustomerName VARCHAR(70) NOT NULL,
+CustomerPhone CHAR(10) NOT NULL,
+CustomerEmail VARCHAR(60) NOT NULL,
+CustomerAge INT ,
+CustomerGender CHAR(1));
+
+CREATE TABLE Address(
+AddressID INT PRIMARY KEY,
```

Comment:
if we have separate table for storing address, then it should have a column to indicate address type, right ? like  work, office